README.md

```diff
@@ -20,9 +20,9 @@ In order to setup and deploy this application, you will need:
 - Clone this repo.
   - `git clone git@github.com:googlegenomics/beacon-go.git`
 - Edit the configuration.
-  - In `beacon.go`, edit the value of `variantSetIds` to reference your data.
+  - In `app.yaml`, edit the values of `GOOGLE_CLOUD_PROJECT` and `GOOGLE_BIGQUERY_TABLE` to reference your data.
 - Deploy.
-  - `goapp deploy -application <project name> app.yaml`
+  - `gcloud app deploy app.yaml --project <project name>`
```

Comment:
@kemp-google, can you please review this line before I merge, it has been changed in a parallel MR and I had to resolve the conflict